### PR TITLE
CalDav support

### PR DESCRIPTION
**Pull request by Dimonyga**

### CODE REVIEW

File: thunderbird.php [hunk 1]
```diff
@@ -14,13 +15,14 @@
 ldap_set_option($link, LDAP_OPT_PROTOCOL_VERSION, 3);
 $ldap_bind = ldap_bind($link, $ldap_user, $ldap_password);
 
+
```
Comment (k0ste) [inline thunderbird.php]: Remove extra whitespace.

File: ldap.defaults.php [hunk 2]
```diff
@@ -5,4 +5,18 @@
 $ldap_sn = '';
 $ldap_title = '';
 $ldap_mail = '';
+$caldav_fillconf = '';
```
Comment (k0ste) [inline ldap.defaults.php]: This is no longer necessary.

**Comment by Dimonyga** [comment]
Fixed

**Comment by k0ste** [comment]
Thanks.

**Pull request merged**